core/src/main/resources/lib/layout/badge.jelly

```diff
@@ -39,7 +39,7 @@ THE SOFTWARE.
     </st:attribute>
   </st:documentation>
   <j:if test="${attrs.badge != null}">
-    <span class="jenkins-badge alert-${attrs.badge.severity} ${attrs.class?:''}" tooltip="${attrs.badge.tooltip}">
+    <span class="jenkins-badge jenkins-!-${attrs.badge.severity}-color ${attrs.class?:''}" tooltip="${attrs.badge.tooltip}">
```

Comment:
What's the semantic difference between the two classes here?

Reply:
The alert modifier class feels misused here as the badge isn't an alert (hence I missed this in the alert changes MR). Meanwhile the colour classes are relatively widely used in Jenkins already and are generic classes to modify colours of an element, button behaves the same way.